src/sentry/static/sentry/app/components/circleIndicator.tsx

```diff
@@ -35,7 +43,7 @@ class CircleIndicator extends React.Component {
     size: 14,
   };
 
-  constructor(props) {
+  constructor(props: Props) {
```

Comment:
Can we just drop this constructor?

Reply:
In fact ideally this component should just be a Styled component.

Reply:
@EvanPurkhiser Yea, I can do that if you think that makes sense